tests/tck/features/expression/MapProjection.feature

```diff
@@ -0,0 +1,177 @@
+      | {v2: ("Tracy McGrady" :player{age: 39, name: "Tracy McGrady"})}                                                   |
+      | {v2: ("Yao Ming" :player{age: 38, name: "Yao Ming"})}                                                             |
+      | {v2: ("JaVale McGee" :player{age: 31, name: "JaVale McGee"})}                                                     |
+      | {v2: ("Tim Duncan" :bachelor{name: "Tim Duncan", speciality: "psychology"} :player{age: 42, name: "Tim Duncan"})} |
+
+  Scenario: All-properties selector
+    Given a graph with space named "nba"
+    When executing query:
+      """
+      MATCH (v:player)-[:like]-(v2)
+      WHERE v.age > 45
+      RETURN v {.*}
+      """
+    Then the result should be, in any order, with relax comparison:
+      | v                                  |
+      | {age: 46, name: "Grant Hill"}      |
+      | {age: 46, name: "Grant Hill"}      |
+      | {age: 47, name: "Shaquile O'Neal"} |
+      | {age: 47, name: "Shaquile O'Neal"} |
+      | {age: 47, name: "Shaquile O'Neal"} |
+
+  Scenario: Use a combination of Property selector, Literal entry, Variable selector, All-properties selector
+    Given a graph with space named "nba"
+    When executing query:
+      """
+      MATCH (v:player)-[:like]-(v2)
+      WHERE v.age > 45
+      RETURN v {.name, .*, middleaged: true, v2, playersHeLike: v2 {.*}}
+      """
+    Then the result should be, in any order, with relax comparison:
+      | v                                                                                                                                                                                                                                             |
+      | {age: 46, middleaged: true, name: "Grant Hill", playersHeLike: {age: 39, name: "Tracy McGrady"}, v2: ("Tracy McGrady" :player{age: 39, name: "Tracy McGrady"})}                                                                               |
+      | {age: 46, middleaged: true, name: "Grant Hill", playersHeLike: {age: 39, name: "Tracy McGrady"}, v2: ("Tracy McGrady" :player{age: 39, name: "Tracy McGrady"})}                                                                               |
+      | {age: 47, middleaged: true, name: "Shaquile O'Neal", playersHeLike: {age: 38, name: "Yao Ming"}, v2: ("Yao Ming" :player{age: 38, name: "Yao Ming"})}                                                                                         |
+      | {age: 47, middleaged: true, name: "Shaquile O'Neal", playersHeLike: {age: 31, name: "JaVale McGee"}, v2: ("JaVale McGee" :player{age: 31, name: "JaVale McGee"})}                                                                             |
+      | {age: 47, middleaged: true, name: "Shaquile O'Neal", playersHeLike: {age: 42, name: "Tim Duncan", speciality: "psychology"}, v2: ("Tim Duncan" :bachelor{name: "Tim Duncan", speciality: "psychology"} :player{age: 42, name: "Tim Duncan"})} |
+    # [bug]agg executed failed here now
```

Comment:
Will fix in this pr?

Reply:
I will make another pr to fix this problem